Caboodle/Sms/Sms.shared.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Sms
+    {
+        public static bool CanSendSms => GetCanSendSms();
+
+        public static bool CanSendSmsInBackground => GetCanSendSmsInBackground();
+
+        public static void SendSms(string recipient = null, string message = null)
+        {
+            if (recipient == null)
+                throw new ArgumentNullException(nameof(recipient));
+
+            if (message == null)
+                throw new ArgumentNullException(nameof(message));
+
+            if (CanSendSms)
+                PlatformSendSms(recipient, message);
```

Comment:
I think this should throw if it can't send, not fail silently.

Reply:
Already throwing on platform code.